### PR TITLE
changed to new images

### PR DESCRIPTION
**Pull request by fatz**
universe images got replaced with new versions

### CODE REVIEW

File: repo/packages/A/alluxio-enterprise/0/resource.json [hunk 1]
```diff
@@ -15,33 +15,48 @@
...
+            }
+          ],
+          "kind": "executable",
+          "url": "https://s3-us-west-1.amazonaws.com/alluxio-dcos/autodelete7d/alluxio-enterprise/20170401-161119-g498F5LL42PHGa0i/dcos-alluxio-enterprise.exe"
         }
       }
     }
   }
-}
+}
```
Comment (jsancio) [inline repo/packages/A/alluxio-enterprise/0/resource.json]: Looks like this comment applies to every file. Let's add a newline at the end of the file.

**Review by jsancio — COMMENTED** [review]
First pass looks good. One minor comment.

I am going to test this change using the Universe Server.

**Comment by fatz** [comment]
sorry for the missing new line. So every touched file ends now with a new line

**Comment by jsancio** [comment]
@fatz is this ready to merge? I ask because the description starts with WIP.

**Comment by fatz** [comment]
@jsancio sure. Sorry simply forgot to change that tile. Please go ahead

**Pull request merged**